src/encapsulationKind.ts

```diff
@@ -1,6 +1,38 @@
+// From <https://www.omg.org/spec/DDS-XTypes/1.2/PDF>
```

Comment:
Maybe name the file `EncapsulationKind` to match the casing of the enum? Seems like a thing I've seen code bases do.